### ISSUE TITLE
HTML comment is being escaped

### ISSUE THREAD

**Issue by chrisdpeters** [meldium/breach-mitigation-rails#4]
Using Rails 3.2.13 on Heroku, I _see_ this text at the bottom of every page:

```
<!-- This is a random-length HTML comment: ybfpkxawtmicudovnlgqjerzhs -->
```

It looks like it is being escaped in the source:

```
&lt;!-- This is a random-length HTML comment: ybfpkxawtmicudovnlgqjerzhs --&gt;
```

**Comment by bradleybuda** [comment]
Should be fixed now with #4 - thanks for the bug report.

**Comment by chrisdpeters** [comment]
Thank YOU for fixing it. Wish I could have submitted a pull request along with the report.

Will give it a shot tomorrow for sure!
